ts/client/src/accounts/mangoAccount.ts

```diff
@@ -231,9 +231,24 @@ export class MangoAccount {
    * @param bank
    * @returns native balance for a token, is signed
    */
-  public getTokenBalance(bank: Bank): I80F48 {
+  public getTokenBalance(group: Group, bank: Bank): I80F48 {
```

Comment:
I think having this kind of function is needed, but it should probably be separate from getTokenBalance? Existing users (such as the force-close) could break by changing these semantics.

Reply:
done, I now have a new function called getEffectiveTokenBalance, and I have reset'ted the old getTokenBalance, I think in most cases the UI wouldn't always use getTokenBalance, getEffectiveTokenBalance is what I use now for health, risk, and various getMax* function families